lighthouse/tests/beacon_node.rs

```diff
@@ -52,6 +54,17 @@ struct CommandLineTest {
 }
 impl CommandLineTest {
     fn new() -> CommandLineTest {
+        let mut base_cmd = base_cmd();
+
+        base_cmd
+            .arg("--execution-endpoint")
+            .arg(DEFAULT_EXECUTION_ENDPOINT)
+            .arg("--execution-jwt-secret-key")
+            .arg(DEFAULT_EXECUTION_JWT_SECRET_KEY);
+        CommandLineTest { cmd: base_cmd }
+    }
+
+    fn new_with_no_execution_endpoint() -> CommandLineTest {
```

Comment:
I'd suggest to put some comment here about how to choose between `new_with_no_execution_endpoint` and `new` function?